BigriverBookstore-apple.xcodeproj/xcuserdata/viorelCornel.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@c-Viorel Let's add this file to gitignore, since is specific to your local Xcode.

Reply:
@c-Viorel This file is still on the repo, don't forget about gitignore and deleting this file from repo.